sensor.js

```diff
@@ -236,6 +242,22 @@ Rtl433Accessory.prototype = {
           }
 
           break;
+        case "contact":
+	  // {"time" : "2022-03-17 23:04:08.430623", "protocol" : 68, "model" : "Kerui-Security", "id" : 297536, "cmd" : 14, "opened" : 1, "state" : "open" }
+	  // {"time" : "2022-03-17 23:04:19.447761", "protocol" : 68, "model" : "Kerui-Security", "id" : 297536, "cmd" : 7, "opened" : 0, "state" : "close" }
+          // debug("update door this--->",this);
+          // console.log("update door this--->",this);
+
+          this.sensorService
+              .setCharacteristic(Characteristic.ContactSensorState, data.opened );
+
+          if (data.battery !== undefined || data.battery_ok != undefined) {
+            var batteryOk = data.battery === "OK" || data.battery_ok === 1
+            this.sensorService
+              .setCharacteristic(Characteristic.StatusLowBattery, batteryOk ? Characteristic.StatusLowBattery.BATTERY_LEVEL_NORMAL : Characteristic.StatusLowBattery.BATTERY_LEVEL_LOW);
```

Comment:
This can be simplified as

```
.setCharacteristic(Characteristic.StatusLowBattery, (data.battery === "OK" || data.battery_ok === 1 ? Characteristic.StatusLowBattery.BATTERY_LEVEL_NORMAL : Characteristic.StatusLowBattery.BATTERY_LEVEL_LOW));
```

Reply:
ok, i just cut/paste the previous code from case 'humidity' ;-)